CHANGES.rst

```diff
@@ -22,7 +22,8 @@ Bug Fixes
 
 - ``ccd_process`` now copies the meta of the input when subtracting the
   master bias. [#404]
-
```

Comment:
actually I think this PR doesn't need a changelog entry at all because the `location` functionality hasn't been published yet.